squigglepy/distributions.py

```diff
@@ -1,12 +1,14 @@
 import operator
 import math
 import numpy as np
-from matplotlib import pyplot as plt
 from scipy import stats
 
-from .utils import _process_weights_values, _is_numpy, is_dist, _round
+from .utils import _process_weights_values, _is_numpy, is_dist, _round, _optional_import
 from .version import __version__
 
+# We only import matplotlib.pyplot if we need it
+plt = _optional_import("matplotlib.pyplot")
```

Comment:
Do you want to only evaluate this within the plot function?

Reply:
It's not necessary; the module only gets imported when it's actually used.

Reply:
nice